sdk/translation/azure-ai-translation-document/README.md

```diff
@@ -86,6 +86,35 @@ credential = AzureKeyCredential("<api_key>")
 document_translation_client = DocumentTranslationClient(endpoint, credential)
 ```
 
+#### Create the client with an Azure Active Directory credential
+
+`AzureKeyCredential` authentication is used in the examples in this getting started guide, but you can also
+authenticate with Azure Active Directory using the [azure-identity][azure_identity] library.
+Note that regional endpoints do not support AAD authentication. Create a [custom subdomain][custom_subdomain]
+name for your resource in order to use this type of authentication.
+
+To use the [DefaultAzureCredential][default_azure_credential] type shown below, or other credential types provided
+with the Azure SDK, please install the `azure-identity` package:
+
+```pip install azure-identity```
+
+You will also need to [register a new AAD application and grant access][register_aad_app] to
+Form Recognizer by assigning the `"Cognitive Services User"` role to your service principal.
```

Comment:
reference to form recognizer.

On a side note, is it true the service also needs this role?

Reply:
My understanding is that AAD is managed the same for all cognitive services. The docs here [apply](https://docs.microsoft.com/en-us/azure/cognitive-services/authentication?tabs=powershell#authenticate-with-azure-active-directory) and recommend this role type.